views/layouts/main.handlebars

```diff
@@ -12,17 +12,22 @@
   {{#if trackerId}}
   {{> analytics trackerId=trackerId}}
   {{/if}}
+
+  <meta name="defaultLanguage" content="en-US">
+  <meta name="availableLanguages" content="en-US">
+  <link rel="localization" href="locales/send.{locale}.ftl">
```

Comment:
nit: leading `/` on `/locales/...` to be consistent with other absolute asset URLs.